django/src/rdwatch/models/site_evaluation.py

```diff
@@ -179,9 +171,12 @@ def bulk_create_from_region_model(
 
         site_evals: list[SiteEvaluation] = []
         with transaction.atomic():
-            region = get_or_create_region(
-                region_feature.properties.region_id, region_feature.parsed_geometry
-            )[0]
+            # Update the region of the configuration.
+            # If the region doens't have a geometry,
+            # the one from the region model is used.
+            ModelRun.objects.filter(pk=configuration.pk).update(
+                region=get_or_create_region(region_feature.properties.region_id)[0]
```

Comment:
I think you need to add the `region_feature.parsed_geometry as the second argument to the `get_or_create_region` function

Reply:
:facepalm: fixed https://github.com/ResonantGeoData/RD-WATCH/pull/396/commits/e556ca4aa6a2ec7d5e72196fb94eb5c0d02ea5a4